support detecting MSVC v142 - VS 2019 C++ build tools v14.2

On Windows, when adding `.linkSystemLibrary("c");` to my `build.zig` I get the following errors. 
```
lld: error: could not open 'libcmtd.lib': no such file or directory
lld: error: could not open 'libvcruntimed.lib': no such file or directory
lld: error: could not open 'legacy_stdio_definitions.lib': no such file or directory
```

I assume this is because I updated my visual studio installation from 16.4.something (I think it was .4) to 16.5.0. I also updated my zig before trying to compile my project where I link against C libraries. I tried downgrading to what I think was my previous zig version, but it didn't make any difference.

I'm currently at zig 0.5.0+555a2c032

Any ideas on what might be wrong or how to fix it are welcome. 

I do have the libraries in: `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.25.28610\lib\x64`

I have the same issue as well. This used to work before either the zig cc branch or the llvm-10 branch

My build.zig
```zig
const Builder = @import("std").build.Builder;

pub fn build(b: *Builder) void {
    const mode = b.standardReleaseOptions();

    const bindings = b.addStaticLibrary("wwiseBindings", null);
    bindings.linkSystemLibrary("c");
    bindings.linkSystemLibrary("AkSoundEngine");
    bindings.linkSystemLibrary("AkStreamMgr");
    bindings.linkSystemLibrary("AkMemoryMgr");
    bindings.linkSystemLibrary("CommunicationCentral");
    bindings.linkSystemLibrary("AkRoomVerbFX");
    bindings.linkSystemLibrary("AkStereoDelayFX");
    bindings.linkSystemLibrary("AkVorbisDecoder");
    bindings.linkSystemLibrary("ole32");
    bindings.linkSystemLibrary("user32");
    bindings.linkSystemLibrary("advapi32");
    bindings.linkSystemLibrary("ws2_32");
    bindings.addIncludeDir("bindings/IOHook/Win32");
    bindings.addIncludeDir("WwiseSDK/include");
    bindings.addLibPath("WwiseSDK/x64_vc150/Profile(StaticCRT)/lib");

    const bindingsSources = &[_][]const u8{
        "bindings/zig_wwise.cpp",
        "bindings/IOHook/Common/AkFilePackage.cpp",
        "bindings/IOHook/Common/AkFilePackageLUT.cpp",
        "bindings/IOHook/Common/AkMultipleFileLocation.cpp",
        "bindings/IOHook/Win32/AkDefaultIOHookBlocking.cpp",
    };
    for (bindingsSources) |src| {
        bindings.addCSourceFile(src, &[_][]const u8{ "-std=c++17", "-DUNICODE" });
    }

    const imgui = b.addStaticLibrary("zigimgui", null);
    imgui.linkSystemLibrary("c");

    const imguiSources = &[_][]const u8{
        "imgui/cimgui.cpp",
        "imgui/imgui/imgui.cpp",
        "imgui/imgui/imgui_demo.cpp",
        "imgui/imgui/imgui_draw.cpp",
        "imgui/imgui/imgui_impl_dx11.cpp",
        "imgui/imgui/imgui_impl_win32.cpp",
        "imgui/imgui/imgui_widgets.cpp",
    };
    for (imguiSources) |src| {
        imgui.addCSourceFile(src, &[_][]const u8{"-std=c++17"});
    }

    const exe = b.addExecutable("wwiseZig", "src/main.zig");
    exe.setBuildMode(mode);
    exe.addIncludeDir("bindings");
    exe.addIncludeDir("imgui");
    exe.addIncludeDir("WwiseSDK/include");
    exe.addLibPath("WwiseSDK/x64_vc150/Profile(StaticCRT)/lib");
    exe.linkLibrary(bindings);
    exe.linkLibrary(imgui);
    exe.linkSystemLibrary("c");
    exe.linkSystemLibrary("D3D11");
    exe.linkSystemLibrary("dxguid");
    exe.install();

    const run_cmd = exe.run();
    run_cmd.step.dependOn(b.getInstallStep());

    const run_step = b.step("run", "Run the app");
    run_step.dependOn(&run_cmd.step);
}

```

```
C:\Programmation\Zig\zig\out\install\x64-Release\bin\zig.exe build-lib --library c --library AkSoundEngine --library AkStreamMgr --library AkMemoryMgr --library CommunicationCentral --library AkRoomVerbFX --library AkStereoDelayFX --library AkVorbisDecoder --library ole32 --library user32 --library advapi32 --library ws2_32 --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\zig_wwise.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Common\AkFilePackage.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Common\AkFilePackageLUT.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Common\AkMultipleFileLocation.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Win32\AkDefaultIOHookBlocking.cpp --cache-dir C:\Programmation\Zig\WwiseDemo\zig-cache --name wwiseBindings -I C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Win32 -I C:\Programmation\Zig\WwiseDemo\WwiseSDK\include -L WwiseSDK/x64_vc150/Profile(StaticCRT)/lib --cache on 
In file included from C:\Programmation\Zig\WwiseDemo\bindings\zig_wwise.cpp:6:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/SoundEngine/Common/AkSoundEngine.h:39:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/SoundEngine/Common/IAkPlugin.h:34:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/SoundEngine/Common/AkCommonDefs.h:38:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/SoundEngine/Common/AkSpeakerVolumes.h:41:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/Tools/Common/AkPlatformFuncs.h:37:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/SoundEngine/Common/AkAtomic.h:33:
In file included from C:\Programmation\Zig\WwiseDemo\WwiseSDK\include\AK/SoundEngine/Platforms/Windows/AkAtomic.h:32:
C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\ucrt\..\um\windows.h:167:10: fatal error: 'excpt.h' file not found
#include <excpt.h>
         ^~~~~~~~~
1 error generated.

The following command failed:
C:\Programmation\Zig\zig\out\install\x64-Release\bin\zig.exe clang -c -MD -MV -MF C:\Programmation\Zig\WwiseDemo\zig-cache\tmp\cKXEf5tuwT9_-zig_wwise.obj.d -nostdinc -fno-spell-checking -isystem C:\Programmation\Zig\zig\lib\include -isystem C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\ucrt -isystem C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\lib\x64\..\..\include -isystem C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\ucrt\..\um -isystem C:\Program Files (x86)\Windows Kits\10\Include\10.0.18362.0\ucrt\..\shared -target x86_64-unknown-windows-msvc -Xclang -target-cpu -Xclang skylake -Xclang -target-feature -Xclang -3dnow,-3dnowa,+64bit,+adx,+aes,+avx,+avx2,-avx512bf16,-avx512bitalg,-avx512bw,-avx512cd,-avx512dq,-avx512er,-avx512f,-avx512ifma,-avx512pf,-avx512vbmi,-avx512vbmi2,-avx512vl,-avx512vnni,-avx512vp2intersect,-avx512vpopcntdq,+bmi,+bmi2,-branchfusion,-cldemote,+clflushopt,-clwb,-clzero,+cmov,+cx16,+cx8,-enqcmd,+ermsb,+f16c,-false-deps-lzcnt-tzcnt,+false-deps-popcnt,-fast-11bytenop,-fast-15bytenop,-fast-bextr,+fast-gather,-fast-hops,-fast-lzcnt,+fast-scalar-fsqrt,-fast-scalar-shift-masks,+fast-shld-rotate,+fast-variable-shuffle,+fast-vector-fsqrt,-fast-vector-shift-masks,+fma,-fma4,+fsgsbase,+fxsr,-gfni,-idivl-to-divb,+idivq-to-divl,+invpcid,-lea-sp,-lea-uses-ag,-lwp,+lzcnt,+macrofusion,+merge-to-threeway-branch,+mmx,+movbe,-movdir64b,-movdiri,-mpx,-mwaitx,+nopl,-pad-short-functions,+pclmul,-pconfig,-pku,+popcnt,-prefer-128-bit,-prefer-256-bit,-prefer-mask-registers,-prefetchwt1,+prfchw,-ptwrite,-rdpid,+rdrnd,+rdseed,-retpoline,-retpoline-external-thunk,-retpoline-indirect-branches,-retpoline-indirect-calls,-rtm,+sahf,+sgx,-sha,-shstk,+slow-3ops-lea,-slow-incdec,-slow-lea,-slow-pmaddwd,-slow-pmulld,-slow-shld,-slow-two-mem-ops,-slow-unaligned-mem-16,-slow-unaligned-mem-32,-soft-float,+sse,-sse-unaligned-mem,+sse2,+sse3,+sse4.1,+sse4.2,-sse4a,+ssse3,-tbm,-use-aa,-use-glm-div-sqrt-costs,-vaes,-vpclmulqdq,+vzeroupper,-waitpkg,-wbnoinvd,+x87,-xop,+xsave,+xsavec,+xsaveopt,+xsaves -g -fno-omit-frame-pointer -fsanitize=undefined -fsanitize-trap=undefined -D_DEBUG -Og -fstack-protector-strong --param ssp-buffer-size=4 -I C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Win32 -I C:\Programmation\Zig\WwiseDemo\WwiseSDK\include -o C:\Programmation\Zig\WwiseDemo\zig-cache\tmp\cKXEf5tuwT9_-zig_wwise.obj C:\Programmation\Zig\WwiseDemo\bindings\zig_wwise.cpp -std=c++17 -DUNICODE
wwiseBindings...The following command exited with error code 1:
C:\Programmation\Zig\zig\out\install\x64-Release\bin\zig.exe build-lib --library c --library AkSoundEngine --library AkStreamMgr --library AkMemoryMgr --library CommunicationCentral --library AkRoomVerbFX --library AkStereoDelayFX --library AkVorbisDecoder --library ole32 --library user32 --library advapi32 --library ws2_32 --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\zig_wwise.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Common\AkFilePackage.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Common\AkFilePackageLUT.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Common\AkMultipleFileLocation.cpp --c-source -std=c++17 -DUNICODE C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Win32\AkDefaultIOHookBlocking.cpp --cache-dir C:\Programmation\Zig\WwiseDemo\zig-cache --name wwiseBindings -I C:\Programmation\Zig\WwiseDemo\bindings\IOHook\Win32 -I C:\Programmation\Zig\WwiseDemo\WwiseSDK\include -L WwiseSDK/x64_vc150/Profile(StaticCRT)/lib --cache on 

Build failed. The following command failed:
C:\Programmation\Zig\WwiseDemo\zig-cache\o\Hbms29RtiSyBQIttvYOPrPaHmjsMNL99VYoaf_hiR3Sje98psFYVmDwZ8JYPJ9B2\build.exe C:\Programmation\Zig\zig\out\install\x64-Release\bin\zig.exe C:\Programmation\Zig\WwiseDemo C:\Programmation\Zig\WwiseDemo\zig-cache --verbose
```

--- a/src/codegen.cpp
+++ b/src/codegen.cpp
@@ -9057,80 +9057,13 @@ static void detect_libc(CodeGen *g) {
     if (g->zig_target->is_native_os) {
         g->libc = heap::c_allocator.create<Stage2LibCInstallation>();
 
-        // search for native_libc.txt in following dirs:
-        //   - LOCAL_CACHE_DIR
-        //   - GLOBAL_CACHE_DIR
-        // if not found create at:
-        //   - GLOBAL_CACHE_DIR
-        // be mindful local/global caches may be the same dir
-
-        Buf basename = BUF_INIT;
-        buf_init_from_str(&basename, "native_libc.txt");
-
-        Buf local_libc_txt = BUF_INIT;
-        os_path_join(g->cache_dir, &basename, &local_libc_txt);
-
-        Buf global_libc_txt = BUF_INIT;
-        os_path_join(get_global_cache_dir(), &basename, &global_libc_txt);
-
-        Buf *pathnames[3] = { nullptr };
-        size_t pathnames_idx = 0;
-
-        pathnames[pathnames_idx] = &local_libc_txt;
-        pathnames_idx += 1;
-
-        if (!buf_eql_buf(pathnames[0], &global_libc_txt)) {
-            pathnames[pathnames_idx] = &global_libc_txt;
-            pathnames_idx += 1;
+        if ((err = stage2_libc_find_native(g->libc))) {
+            fprintf(stderr,
+                "Unable to link against libc: Unable to find libc installation: %s\n"
+                "See `zig libc --help` for more details.\n", err_str(err));
+            exit(1);
         }
 
-        Buf* libc_txt = nullptr;
-        for (auto name : pathnames) {
-            if (name == nullptr)
-                break;
-
-            bool result;
-            if (os_file_exists(name, &result) != ErrorNone || !result)
-                continue;
-
-            libc_txt = name;
-            break;
-        }
-
-        if (libc_txt == nullptr)
-            libc_txt = &global_libc_txt;
-
-        if ((err = stage2_libc_parse(g->libc, buf_ptr(libc_txt)))) {
-            if ((err = stage2_libc_find_native(g->libc))) {
-                fprintf(stderr,
-                    "Unable to link against libc: Unable to find libc installation: %s\n"
-                    "See `zig libc --help` for more details.\n", err_str(err));
-                exit(1);
-            }
-            Buf libc_txt_dir = BUF_INIT;
-            os_path_dirname(libc_txt, &libc_txt_dir);
-            buf_deinit(&libc_txt_dir);
-            if ((err = os_make_path(&libc_txt_dir))) {
-                fprintf(stderr, "Unable to create %s directory: %s\n",
-                    buf_ptr(g->cache_dir), err_str(err));
-                exit(1);
-            }
-            Buf *native_libc_tmp = buf_sprintf("%s.tmp", buf_ptr(libc_txt));
-            FILE *file = fopen(buf_ptr(native_libc_tmp), "wb");
-            if (file == nullptr) {
-                fprintf(stderr, "Unable to open %s: %s\n", buf_ptr(native_libc_tmp), strerror(errno));
-                exit(1);
-            }
-            stage2_libc_render(g->libc, file);
-            if (fclose(file) != 0) {
-                fprintf(stderr, "Unable to save %s: %s\n", buf_ptr(native_libc_tmp), strerror(errno));
-                exit(1);
-            }
-            if ((err = os_rename(native_libc_tmp, libc_txt))) {
-                fprintf(stderr, "Unable to create %s: %s\n", buf_ptr(libc_txt), err_str(err));
-                exit(1);
-            }
-        }
         bool want_sys_dir = !mem_eql_mem(g->libc->include_dir,     g->libc->include_dir_len,
                                          g->libc->sys_include_dir, g->libc->sys_include_dir_len);
         size_t want_um_and_shared_dirs = (g->zig_target->os == OsWindows) ? 2 : 0;
@@ -9164,10 +9097,6 @@ static void detect_libc(CodeGen *g) {
             g->libc_include_dir_len += 1;
         }
         assert(g->libc_include_dir_len == dir_count);
-
-        buf_deinit(&global_libc_txt);
-        buf_deinit(&local_libc_txt);
-        buf_deinit(&basename);
     } else if ((g->out_type == OutTypeExe || (g->out_type == OutTypeLib && g->is_dynamic)) &&
         !target_os_is_darwin(g->zig_target->os))
     {

